rkt/rkt.go

```diff
@@ -139,6 +140,30 @@ func getKeystore() *keystore.Keystore {
 	return keystore.New(config)
 }
 
+func getDataDir() string {
+	// If --dir parameter is passed, then use this value.
+	if dirFlag := cmdRkt.PersistentFlags().Lookup("dir"); dirFlag != nil {
+		if dirFlag.Changed {
+			return globalFlags.Dir
+		}
+	} else {
+		panic("Fatal error processing rkt arguments.")
+	}
+
+	// If above fails, then try to get the value from configuration.
+	if config, err := getConfig(); err != nil {
```

Comment:
Hm, this datadir function is called quite often sometimes. I suppose we will need to implement a kind of caching for getting a config in `getConfig` function to avoid doing useless disk IO.